buildSrc/repos.gradle

```diff
@@ -76,6 +75,7 @@ def addMavenRepositories(RepositoryHandler handler) {
                 includeModule("com.android.tools", "r8")
             }
         }
+        handler.mavenLocal()
     }
```

Comment:
Let's have a comment about why it should be here at the end.

Reply:
Done